.github/workflows/build-common.yml

```diff
@@ -302,6 +302,7 @@ jobs:
             **/jitdump.*.dmp
           if-no-files-found: ignore
 
+<<<<<<< HEAD
```

Comment:
This line caused CI running failed, for detail you can refer to https://github.com/open-telemetry/opentelemetry-java-instrumentation/actions/runs/8406624268